static/js/components/tiles/line_tile.tsx

```diff
@@ -125,19 +130,47 @@ export const fetchData = async (props: LineTilePropType) => {
       statVars.push(spec.denom);
     }
   }
-  let endpoint = "/api/observations/series";
-  if (props.apiRoot) {
-    endpoint = props.apiRoot + endpoint;
+
+  let params;
+  let endpoint;
+  if (!_.isEmpty(props.comparisonPlaces)) {
+    endpoint = `${props.apiRoot || ""}/api/observations/series`;
+    params = {
+      entities: props.comparisonPlaces,
+      variables: statVars,
+    };
+  } else if (props.enclosedPlaceType) {
+    endpoint = `${props.apiRoot || ""}/api/observations/series/within`;
+    params = {
+      parentEntity: props.place.dcid,
+      childType: props.enclosedPlaceType,
+      variables: statVars,
+    };
+  } else {
+    endpoint = `${props.apiRoot || ""}/api/observations/series`;
+    params = {
+      variables: statVars,
+      entities: [props.place.dcid],
+    };
   }
+
   const resp = await axios.get(endpoint, {
     // Fetch both numerator stat vars and denominator stat vars
-    params: {
-      variables: statVars,
-      entities: [props.place.dcid],
-    },
+    params,
     paramsSerializer: stringifyFn,
   });
-  return rawToChart(resp.data, props);
+
+  // get place names from dcids
+  const placeDcids = Object.keys(resp.data.data[statVars[0]]);
+  const placeNames = await getPlaceNames(placeDcids, props.apiRoot);
+  // How legend labels should be set
+  const options = {
+    // If many places and one stat var, legend should show only place labels
```

Comment:
add a comment that if neither of these are set, we will show the stat vars in the legend labels?

Reply:
Added!